### DISCUSSION TITLE
More Features for Map Creators

### DISCUSSION THREAD

**Discussion by Mephisto-Fibs** [Keriew/augustus#552] [Ideas]
It would be nice if there were new features for map makers, too.

1.) Unlockable Buildings
It would be useful if certain buildings in the game have to be unlocked first ... for example, the Native Mission could only be unlocked in a certain year, when the greatest battles are fought and the inhabitants of a territory are subjugated.
(This function of the gradually unlockable buildings is already included in the tutorial, where buildings only gradually appear in the construction inventory.)

2. Immensly Important Map Maker Feature: 
I think it should be possible to place and name the city icon on the map of the empire; Furthermore, it would be so great and useful  if it were finally possible to place cities for trading yourself, name them and determine their trading options.

3.) Pre-existing Structures:
I would find it very nice if structures such as bridges, pallisade walls and watchtowers can be set when the map is being drawn up, as these were often built by the Romans long before an area was finally settled.

4.) Small Campaign
It would be great if it were possible to create a small campagne ... a sequence of maps that you have to defeat in a certain order, which build on each other like in the vanilla game. Once a map is won, you are presented with the next one in the sequence. This way, you are able to recreate the historical campain of, say, a general that is conquering a whole area one city at a time. 

 -----  (edited from here on:  added comments after crudelios' reply from Dec. 18)  -----

5.) Mission Briefings
I think a feature for adding instructional input before a custom map starts could be very very useful for map makers. These inputs could be pure text or even voiced audio. These mission briefings pop up before a custom made scenario to tell the player what this scenario is about ((just as in the base game). 
Furthermore, it could be beneficial to be able to make these briefings pop up (also) at any specific time during the scenario... The map maker could be able to schedule a custom made 'letter from Rome' to the governor during the game. This might add a little more immersion into the storyline of the map and enables the map maker to focus the players attention to a specific problem, allowing more story driven and in depth scenarios.

I'm always a sucker for more immersive features. What do you think? Any further Ideas for Features used in the map creation process?

**Comment by crudelios** [comment]
Actually all your suggestions are planned for a future release  🙂

**Comment by thomasherena** [comment]
To add to this, I think it'd be cool to have pre-built settlements, sort of how there were in certain Pharaoh missions.

**Comment by Mephisto-Fibs** [comment]
5.) I think a feature for adding instructional input before a custom map starts could be very useful for map makers. These inputs could be pure text or even voiced audio (just as in the base game).

**Reply by Mephisto-Fibs** [reply]
What I meant was to add mission briefings to custom maps... I just noticed in the Actions that there was a change concerning mission briefings ... Is it now possible to add a vanilla style mission briefing to custom maps? And if so, how?

**Comment by Ranmyaku2** [comment]
Being able to determine where a natural event like the earthquake occurs exactly would be nice. If possible also determining if it can be repaired or not and how long before it might disappear or be fixable extra would be a nice feature.

**Comment by Salve970** [comment]
Be nice to have the option to add New Trading Cities.

**Reply by crudelios** [reply]
This option now exists on the latest unstable actually

**Comment by UberWaffe** [comment]
> 
> 1.) Unlockable Buildings It would be useful if certain buildings in the game have to be unlocked first ... for example, the Native Mission could only be unlocked in a certain year, when the greatest battles are fought and the inhabitants of a territory are subjugated. (This function of the gradually unlockable buildings is already included in the tutorial, where buildings only gradually appear in the construction inventory.)
> 
> 5.) Mission Briefings I think a feature for adding instructional input before a custom map starts could be very very useful for map makers. These inputs could be pure text or even voiced audio. These mission briefings pop up before a custom made scenario to tell the player what this scenario is about ((just as in the base game). Furthermore, it could be beneficial to be able to make these briefings pop up (also) at any specific time during the scenario... The map maker could be able to schedule a custom made 'letter from Rome' to the governor during the game. This might add a little more immersion into the storyline of the map and enables the map maker to focus the players attention to a specific problem, allowing more story driven and in depth scenarios.
> 

Requested feature 1 and 5 are now possible with the recently added custom events and custom messages.
